presto-main/src/main/java/com/facebook/presto/execution/executor/TaskExecutor.java

```diff
@@ -94,11 +100,15 @@ public class TaskExecutor
     private static final Duration LONG_SPLIT_WARNING_THRESHOLD = new Duration(600, SECONDS);
     // Interrupt a split if it is running longer than this AND it's blocked on something known
     private static final Predicate<List<StackTraceElement>> DEFAULT_INTERRUPTIBLE_SPLIT_PREDICATE = elements ->
-                    elements.stream()
-                            .anyMatch(element -> element.getClassName().equals(JoniRegexpFunctions.class.getName()));
+            elements.stream()
+                    .anyMatch(element -> element.getClassName().equals(JoniRegexpFunctions.class.getName()));
     private static final Duration DEFAULT_INTERRUPT_SPLIT_INTERVAL = new Duration(60, SECONDS);
 
     private static final AtomicLong NEXT_RUNNER_ID = new AtomicLong();
+    public static final String OB_WAIT_OVER = "obWait-over";
+    public static final String OB_WAIT = "obWait";
+    public static final String SPLIT_WAIT = "splitWait";
+    public static final String SPLIT_WAIT_OVER = "splitWait-over";
```

Comment:
if we are planning to keep it in prod, might be good to add ordinals prefixed so that it is easier to understand the sequence of events